src/main/java/backend/resource/Model.java

```diff
@@ -99,7 +103,7 @@ public UpdateSignature getUpdateSignature() {
     }
 
     @Override
-    public List<TurboIssue> getIssues() {
+    public synchronized List<TurboIssue> getIssues() {
```

Comment:
Why is this needed? Is it safe to synchronise only this method and not others which access `issues`?